src/api.ts

```diff
@@ -15,6 +17,7 @@ function get<T>(url: string): Promise<T> {
   // eslint-disable-next-line prefer-template
   const fullURL = BASE_URL + url + '.json';
```

Comment:
Consider using template literals for URL concatenation for better readability and to avoid potential errors with string concatenation. For example: `const fullURL = `${BASE_URL}${url}.json`;`